### DISCUSSION TITLE
[Documentation] REST API for permissions is unclear

### DISCUSSION THREAD

**Discussion by VerdonTrigance** [keycloak/keycloak#9888] [Q&A]
Hi everyone.
I read https://www.keycloak.org/docs/latest/authorization_services/ many times, but still didn't get the idea of associating permissions with resource. Should I use https://www.keycloak.org/docs/latest/authorization_services/#associating-a-permission-with-a-resource or https://www.keycloak.org/docs/latest/authorization_services/#creating-permission-ticket-2 ? I tried first one, but faced with naming problem. After creating a resource I'm giving permissions to everyone to it and named it like "Everyone can read", but after I had created the second resource and did same thing I got "Policy with same name already exists" (or something like this). So what should I do? It seems like https://www.keycloak.org/docs/latest/authorization_services/#associating-a-permission-with-a-resource tied with resource, but has global naming scope. So I think it should work as below: 
I create a resource, then create a policy without binding it to resource, and after all I should be able to associate it with multiple resources. That's what I thinking.
Please help me out.

**Comment by dodyg** [comment]
https://www.youtube.com/watch?v=3K77Pvv-ouU
